python/nano/setup.py

```diff
@@ -77,14 +77,15 @@ def setup_package():
                            "keras==2.7.0",
                            "tensorflow-estimator==2.7.0"]
 
+    # ipex is only avaliable for linux now
     pytorch_requires = ["torch==1.11.0",
                         "torchvision==0.12.0",
                         "pytorch_lightning==1.6.4",
                         "torchmetrics==0.7.2",
                         "opencv-python-headless",
                         "PyTurboJPEG",
                         "opencv-transforms",
-                        "intel_extension_for_pytorch==1.11.0"]
+                        "intel_extension_for_pytorch==1.11.0;platform_system!='Windows'"]
```

Comment:
Could you provide the syntex/doc reference that support the platform check logic here in the PR comment?

Reply:
> Could you provide the syntex/doc reference that support the platform check logic here in the PR comment?


You can refer to this doc: https://setuptools.pypa.io/en/latest/userguide/dependency_management.html.
And the syntax is here: https://peps.python.org/pep-0508/